assets/sass/kerrokantasi/_common.scss

```diff
@@ -251,4 +251,18 @@ a.list-group-item {
 
 .icon-adjustment {
   top: $font-size-base;
+}
+
+.progressBar-wrapper {
+  height: 1.4em;
+  position: relative;
+  width: 100%;
+  border-radius: 5px;
+  background: $light-gray;
+  .progressBar {
+    height: 100%;
+    position: relative;
+    border-radius: inherit;
+    background: blue;
```

Comment:
I think should use --color-coat-of-arms to keep original color 🤔

Reply:
Actually the original color comes straight from bootstrap but you're right, I should add Helsinki theme styles and use coat-of-arms there 👍